packages/documentation/docs/pages/Builder.md

```diff
@@ -2,18 +2,24 @@
 
 The [UI5 Builder](https://github.com/SAP/ui5-builder) module takes care of building your project.
 
+<script setup>
+import VPButton from "vitepress/dist/client/theme-default/components/VPButton.vue"
+</script>
+
 Based on a project's type, the UI5 Builder defines a series of build steps to execute; these are also called "tasks".
 
-For every type there is a set of default tasks. You can disable single tasks using the `--exclude-task` [CLI parameter](./CLI.md#ui5-build), and you can include tasks using the `--include-task` parameter.
+For every type there is a set of default tasks. You can disable single tasks using the `--exclude-task` [CLI parameter](./CLI#ui5-build), and you can include tasks using the `--include-task` parameter.
```

Comment:
Is there really no other solution for the links? It's very unfortunate that the relative links won't work anymore when visiting the markdown page on github.